test/IISTestSite/web.config

```diff
@@ -4,6 +4,10 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false" hostingModel="inprocess"/>
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false" hostingModel="inprocess">
+      <environmentVariables>
+        <environmentVariable name="ASPNETCORE_INPROCESS_TESTING_VALUE" value="foobar" />
```

Comment:
how about a test for adding to an existing variable?

Reply:
Done, and a few other tests.